sweetalert2.d.ts

```diff
@@ -385,13 +385,46 @@ declare module 'sweetalert2' {
     footer?: string;
   }
 
+  type Awaited<T> = T extends Promise<infer U> ? U : T;
+
   type SyncOrAsync<T> = T | Promise<T>;
 
   type ValueOrThunk<T> = T | (() => T);
 
+  type UpdatableParameters =
```

Comment:
It would be better to export this to perhaps use it in other packages like ngx-sweetalert2?

Reply:
Agree, please do export `UpdatableParameters`

Reply:
Done :)